direct_message/tests.py

```diff
@@ -1,3 +1,82 @@
-# from django.test import TestCase
+from . import models
+import pytest
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your tests here.
+
+CREATION_DATE = timezone.now()
+
+
+@pytest.fixture
+def new_user_sender():
+    return User(username='TestSender', first_name='Nick',
+                last_name='Birch', password='1234', email='Nick@email.com')
+
+
+@pytest.fixture
+def new_user_receiver():
+    return User(username='TestReceiver', first_name='Andrew',
+                last_name='Glouberman', password='abcd', email='Andrew@email.com')
+
+
+@pytest.fixture
+def new_message(new_user_sender, new_user_receiver):
+    return models.Message(sender=new_user_sender,
+                          receiver=new_user_receiver,
+                          creation_date=CREATION_DATE,
+                          message_text='test')
+
+
+def save_a_message(new_message):
+    new_message.sender.save()
+    new_message.receiver.save()
+    new_message.save()
+
+
+def test_message_constructor(new_message, new_user_sender, new_user_receiver):
```

Comment:
Maybe we can change this test name to be more clear? if this checks a new message maybe we can call something like `test_new_message`?

Reply:
fixed to 'new_message'

Reply:
I tend to agree with Yuli

Reply:
Changed, I also changed a couple of more names to be more consistent with Yuli's suggestion @natifridman  @yulisuliman